### ISSUE TITLE
It's no longer possible to get a link to join Slack

### ISSUE THREAD

**Issue by gosukiwi** [atom/atom#21204]
The README and [Discussion boards](https://discuss.atom.io/t/join-us-on-slack/16638?source_topic_id=25406) say that you can get an invite to Slack by emailing `atom@github.com`. But when you send an email, you get a response:

>Hi there,
We now require that new support requests be created using our Support website:
https://support.github.com
Please use this website to search through resources that may help you find the solutions you are looking for or connect with our GitHub experts.
Thank you,
GitHub Support
**Please do not reply to this email, as it will not be seen by our team.**

Shouldn't the README and Discussion boards be updated?

**Comment by calebmeyer** [comment]
Yep, for some reason I remember just being able to create an account and join the slack directly, but now the signup page requires a specific email:
<img width="1983" alt="image" src="https://user-images.githubusercontent.com/3342777/90827259-8e4b6680-e301-11ea-92dd-041f39488ae5.png">
